Middleware/DynamicJavaScriptMiddleware.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.IO;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Hosting;
+using System.Threading.Tasks;
+
+public class DynamicJavaScriptMiddleware
+{
+    private readonly RequestDelegate _next;
+    private readonly IWebHostEnvironment _env;
+
+    public DynamicJavaScriptMiddleware(RequestDelegate next, IWebHostEnvironment env)
+    {
+        _next = next;
+        _env = env;
+    }
+
+    public async Task InvokeAsync(HttpContext context)
+    {
+        OSFLogo osfLogo = new OSFLogo();
```

Comment:
Echoing Scotts' point, not using var is very old fashioned. I'd suggest introducing it across the board